sources/light_client.move

```diff
@@ -19,6 +19,9 @@ const EHeaderListIsEmpty: u64 = 4;
 const EBlockNotFound: u64 = 5;
 const EForkChainWorkTooSmall: u64 = 6;
 const ETxNotInBlock: u64 = 7;
+const EExtendBeyondFinal: u64 = 8;
+
+const FINALITY: u64 = 8;
```

Comment:
Could we move this in the parameter when creating a light client?

Reply:
OK, moved to LC field